apis/io-engine/src/v1.rs

```diff
@@ -11,7 +11,7 @@ mod pb {
 }
 
 pub mod common {
-    pub use super::pb::{Cipher, Encryption, EncryptionKey, ShareProtocol};
+    pub use super::pb::{Cipher, EncryptionData, EncryptionKey, EncryptionSecret, ShareProtocol};
```

Comment:
Do we need to export something for the oneOf ?

Reply:
Right. Let me do it.